### ISSUE TITLE
Refer-a-Friend Problems

### ISSUE THREAD

**Issue by Garkelem** [dalaranwow/dalaran-wow#3273]
I recently recruited a friend, but even though he created the account through the email we are not receiving the experience boosts.

**Comment by Golganeth** [comment]
Refer-a-friend bonuses work like a charm, so if you have created the account through the mail you received it should be fine.

Requirements to receive XP bonus:
1) You must have your friend in your friend list, and vice versa.
2) You must be close to each other.
3) You must be in party with your friend.
4) You both must be within the same level range.

Closed.

**Comment by Garkelem** [comment]
This is no help whatsoever.  I understand that is how it's supposed to work. What I'm saying is even though we executed the refer-a-friend process correctly, we are not receiving the benefits, hence the bug claim. I sent the referral, he accepted the the email using the code to create his account, but it does not properly show our accounts as linked.

**Comment by Golganeth** [comment]
The feature XP bonuses are working, hence why I closed the bug report.

What do you mean with 'it does not properly show our accounts as linked'? Take into consideration the requirements mentioned above, as well as the fact that if you kill an NPC alone and get 50 XP, you'll get 75 XP with your friend (the XP is divided between you both -> 50/2 = 25x3 = 75).

If even then the issue persists, you can always make an in-game ticket and we'll make sure to check if something went wrong. Make sure to provide account names.

Thanks!

**Comment by Garkelem** [comment]
When I go to the refer-a-friend page on my account management page it says that 1 of my invitations has been accepted, which is another friend that I referred.  With the first friend it worked correctly, and we do receive the bonus experience when near each other and all of that.  Then I referred a second friend, who also created his account through the email that was sent.  But it does not show his invitation as accepted and we do not receive the bonus experience.
